### ISSUE TITLE
No end of game sounds when loosing last life on space maze.

### ISSUE THREAD

**Issue by bakerhillpins** [jessaskey/mhavocpe#173]
If you loose your last life on the space maze, when the fireballs are falling (not while you are moving through it) no end of game sounds are played.

~TBH, it might be that way when flying through the maze too but I didn't loose my last life there.~  Just tested this. No after game sounds when you loose you last man in space maze.

**Comment by jessaskey** [comment]
So on this... what do you mean by end of game sounds? I think the only sounds that might happen after you die the last time is that if you get a high score, then it will play the music for the high score. Actually, looking at the code... maybe I see the issue... I can see that for top 3 score, you get the 'pre-fireworks'... those should make explosion sounds Then when the high score entry table shows, it should play high score music for any top 10 score. From what I see in the code tho, it might only be playing the music for a top 3 score currently, I still need to test this to verify tho.

**Comment by bakerhillpins** [comment]
I tested this last night, No sounds when you loose your last life anywhere in the space maze and have a score in the top 10.  It appears that the music is played for any High Score, not just the top 3 (which get the fireworks) as in a previous game I got HS 4 and it played the theme while I could enter my name. 

Not sure if it's important (I don't think so) but there's no countdown timer for entering initials. It's plenty long (I sat and waited) but just no indication (like the continue/buyin screen does)

**Comment by jessaskey** [comment]
Confirmed there is no countdown timer for high score entry... Im not going to add one right now either unless I get some spare time. Maybe we can talk about the sounds issue as I dont think I understand what the issue is. :-)

**Comment by bakerhillpins** [comment]
Yea, I don't actually think a HS countdown is needed frankly..  Maybe make it like Star Wars and it's timed to last as long as the High Score theme plays?

I'll go down and play a game or two to make sure I have it straight and we can talk about the missing music. I'll probably update here too for posterity.

**Comment by bakerhillpins** [comment]
So I went and tried this again and I can't make it occur. As you've reported that there were still issues with end of game stuff getting botched up (from the fireworks IIRC) I'm now suspicious that it might have something do do with that. So I guess I should clear the HS table and see what happens. I know I noticed it after a decently long play session so maybe I just need to play for a few hours.
